approach.tex

```diff
@@ -227,83 +174,70 @@ \subsubsection{Test Documents}
-Only then the document can be regenerated with the approved test cases and uploaded in Docushare.
+New or updated test cases shall be in status ``draft''. After the test specification has been approved via RFC, their status need to be set to ``approved''.
+Only then the document can be regenerated and uploaded in Docushare.
 
 The approval of the test plan and report, is a bit different.
 For practical reasons, the test report is completed in 2 steps:
 
 \begin{itemize}
-\item before the test execution: the plan and few information in the test cycle in jira shall be provide, in order to demonstrate that the test is ready to start.
-\item after the test execution: the document is finalized including the reports and comments on the execution of the test cases will, issues raised during the test campaign  and the a statement summarizing the global test campaign result.
+\item before the test execution: the test plan and the test cycle should be created in Jira, in order to demonstrate that the test is ready to start.
+\item after the test execution: the document is finalized including the reports and comments on the execution of the test cases, issues raised during the test campaign and the a statement summarizing the global test campaign result.
 \end{itemize}
 
-Despite only the final document is relevant for deriving the global coverage of the DM requirements, it is important that both steps are consolidated and approved separately.
-Going to execute the test campaign without ensuring that everything is ready for it, it may imply the failure of the test campaign.
+Only the final document is relevant for deriving the global coverage of the DM requirements, but it is important that both steps are approved separately. In this way we ensure all conditions are met to  execute the tests.
 
-The approval of the first step, comes after verification from the product owner, that all needed information are provided in Jira and therefore in the test plan and report document.
-The formality to issue the document in Docushare can be skip, but a coherent draft has to be available in DMTR-XXX.lsst.io and the status of the corresponding test plan has to be set to "approved".
-Stakeholders that may have some expectation on the activity results, like scientists, T/CAMs or others, can be involved before the approval.
+The approval of the first step comes after verification,  by the product owner, that all needed information is provided in Jira and therefore in the test plan and report document.
+The formality of  issuing  the document in Docushare can be skipped, but a coherent draft has to be available in DMTR-XXX.lsst.io and the status of the corresponding test plan has to be set to "approved".
+% In Jira ???
+Stakeholders that may have some expectation on the activity results, like scientists, T/CAMs or others, should  be involved before the approval.
+% SO we are using RFC ??
```

Comment:
No RFC needed in principle. Using the life cycle of the Test Plan defined in Jira should be sufficient.
But we can let open the door to use it.